baus/summaries.py

```diff
@@ -1389,14 +1389,16 @@ def zone_forecast_inputs():
                        index_col="zone_id")
 
 
-def add_population(df, year, regional_controls):
+def add_population(df, year, regional_controls, check_close=True):
     rc = regional_controls
     target = rc.totpop.loc[year] - df.gqpop.sum()
-
     zfi = zone_forecast_inputs()
     s = df.tothh * zfi.meanhhsize
 
-    s = scale_by_target(s, target, .15)
+    if check_close:
+        s = scale_by_target(s, target, .15)
+    else:
+        s = scale_by_target(s, target)
```

Comment:
Update: I now see your comment on this. It looks like you relaxed the range around the target for TAZs. I don't remember running into errors with the population totals before, but it could have happened. It makes me wonder if it has to do with the distribution of household sizes. Could you remove this code and I will note it for the future?

Reply:
Done